packages/template-retail-react-app/app/hooks/use-multiship.js

```diff
@@ -846,26 +476,31 @@ export const useMultiship = (basket) => {
 
     return {
         assignDefaultShippingMethodsToShipments,
-        handleDeliveryOptionChange,
+        updateDeliveryOption,
         removeEmptyShipments,
-        findExistingDeliveryShipment,
-        findExistingPickupShipment,
+        findExistingDeliveryShipment: (basket) =>
+            findExistingDeliveryShipment(basket, isCurrentShippingMethodPickup),
+        findExistingPickupShipment: (basket, storeId) =>
+            findExistingPickupShipment(basket, storeId, isCurrentShippingMethodPickup),
         createNewDeliveryShipment,
         createNewDeliveryShipmentWithAddress,
         createNewPickupShipment,
-        moveItemToDeliveryShipment,
-        moveItemsToDeliveryShipment,
-        moveItemToPickupShipment,
-        moveItemsToPickupShipment,
-        findDeliveryShipmentWithSameAddress,
-        findDeliveryShipmentWithoutAddress,
+        updateItemToDeliveryShipment,
+        updateItemsToDeliveryShipment,
+        updateItemToPickupShipment,
+        updateItemsToPickupShipment,
+        findDeliveryShipmentWithSameAddress: (basket, address) =>
+            findDeliveryShipmentWithSameAddress(basket, address, isCurrentShippingMethodPickup),
+        findDeliveryShipmentWithoutAddress: (basket) =>
+            findDeliveryShipmentWithoutAddress(basket, isCurrentShippingMethodPickup),
         findOrCreateDeliveryShipment,
         findOrCreatePickupShipment,
         getShipmentForItems,
         findEmptyShipments,
         findShipmentToConsolidate,
         getItemsForShipment,
-        findUnusedDeliveryShipment,
+        findUnusedDeliveryShipment: (basket, usedShipmentIds = []) =>
+            findUnusedDeliveryShipment(basket, usedShipmentIds, isCurrentShippingMethodPickup),
         updateDeliveryAddressForShipment,
         areAddressesEqual
     }
```

Comment:
Let's not reexport the methods from other hooks, let's the components use the hook directly if it needs to. We don't want this see this case if someone accidentally do this
```
import {updateItemToDeliveryShipment} from "./multi-ship"
import {updateItemToDeliveryShipment} from './item-arrangement"
``

multiship should export the methods it declares